### PR TITLE
Updating Commons-net

### PR DESCRIPTION
**Pull request by jairmyree**
# Description
The following external dependency has been updated:

- commons-net:commons-net 
  - 3.6 -> 3.9.0

### CODE REVIEW

File: eng/versioning/external_dependencies.txt [hunk 1]
```diff
@@ -160,7 +160,7 @@ org.springframework:spring-jdbc;5.3.24
 org.springframework:spring-jms;5.3.24
 org.springframework:spring-messaging;5.3.24
 org.springframework:spring-tx;5.3.24
-org.springframework:spring-web;5.3.24
+org.springframework:spring-web;6.0.0
```
Comment (alzimmermsft) [inline eng/versioning/external_dependencies.txt:163]: We'll want to wait for the Spring team to make this change as `6.0.0` and later begins requiring Java 17 as the baseline version.

File: eng/versioning/external_dependencies.txt [hunk 2]
```diff
@@ -243,7 +243,7 @@ org.junit.vintage:junit-vintage-engine;5.9.1
 org.openjdk.jmh:jmh-core;1.22
 org.openjdk.jmh:jmh-generator-annprocess;1.22
 org.spockframework:spock-core;2.3-groovy-4.0
-org.testng:testng;7.5
+org.testng:testng;7.7.1
```
Comment (alzimmermsft) [inline eng/versioning/external_dependencies.txt:246]: There is an on-going discussion around this upgrade as TestNG 7.6 and later requires Java 11 as a baseline

Comment (jairmyree) [reply eng/versioning/external_dependencies.txt:246]: Ok that makes sense.

**Review by alzimmermsft — APPROVED** [review]

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**